projects/storefrontlib/src/cms-components/navigation/search-box/search-box.component.html

```diff
@@ -18,95 +23,124 @@
+    tabindex="0"
+  >
+    <li *ngFor="let suggestion of result.suggestions">
+      <a
+        [innerHTML]="suggestion | cxHighlight: searchInput.value"
+        [routerLink]="
+          {
+            cxRoute: 'search',
+            params: { query: suggestion }
+          } | cxUrl
+        "
+        (keydown.arrowup)="focusPreviousChild($event)"
+        (keydown.arrowdown)="focusNextChild($event)"
+        (keydown.enter)="close($event, true)"
+        (keydown.escape)="close($event, true)"
+        (blur)="close($event)"
+        (mousedown)="preventDefault($event)"
+        (click)="
+          dispatchSuggestionEvent({
+            freeText: searchInput.value,
+            selectedSuggestion: suggestion,
+            searchSuggestions: result.suggestions
+          });
+          updateChosenWord(suggestion)
+        "
+      >
+      </a>
+    </li>
+  </ul>
 
-  <div class="products" *ngIf="result.products">
-    <a
-      *ngFor="let product of result.products"
-      [routerLink]="
-        {
-          cxRoute: 'product',
-          params: product
-        } | cxUrl
-      "
-      [class.has-media]="config.displayProductImages"
-      (keydown.arrowup)="focusPreviousChild($event)"
-      (keydown.arrowdown)="focusNextChild($event)"
-      (keydown.enter)="close($event, true)"
-      (keydown.escape)="close($event, true)"
-      (blur)="close($event)"
-      (mousedown)="preventDefault($event)"
-      (click)="
-        dispatchProductEvent({
-          freeText: searchInput.value,
-          productCode: product.code
-        })
-      "
-    >
-      <cx-media
-        *ngIf="config.displayProductImages"
-        [container]="product.images?.PRIMARY"
-        format="thumbnail"
-        [alt]="product.summary"
-      ></cx-media>
-      <h4 class="name" [innerHTML]="product.nameHtml"></h4>
-      <span class="price">{{ product.price?.formattedValue }}</span>
-    </a>
+  <ul
+    class="products"
+    *ngIf="result.products"
+    attr.aria-label="{{ 'searchBox.ariaLabelProducts' | cxTranslate }}"
+    tabindex="0"
+  >
+    <li *ngFor="let product of result.products">
+      <a
+        [routerLink]="
+          {
+            cxRoute: 'product',
+            params: product
+          } | cxUrl
+        "
+        [class.has-media]="config.displayProductImages"
+        (keydown.arrowup)="focusPreviousChild($event)"
+        (keydown.arrowdown)="focusNextChild($event)"
+        (keydown.enter)="close($event, true)"
+        (keydown.escape)="close($event, true)"
+        (blur)="close($event)"
+        (mousedown)="preventDefault($event)"
+        (click)="
+          dispatchProductEvent({
+            freeText: searchInput.value,
+            productCode: product.code
+          })
+        "
+      >
+        <cx-media
+          *ngIf="config.displayProductImages"
+          [container]="product.images?.PRIMARY"
+          format="thumbnail"
+          alt="test"
```

Comment:
I think this change was done by mistake, please revert it
```suggestion
          [alt]="product.summary"
```